eks-worker-al2.json

```diff
@@ -131,6 +131,10 @@
         "ENABLE_FIPS_MODE={{user `enable_fips_mode`}}",
       ]
     },
+    {
+      "type": "shell",
+      "inline": ["sudo reboot"]
```

Comment:
We'll need `expect_disconnect` set to `true`, and we should move the `pause_before` from the next provisioner to a `pause_after` in this provisioner, if possible.

Reply:
Done, and I also removed the `expect_disconnect` from the `upgrade_kernel.sh` provisioner as there's no longer a reboot residing in that script.